libs/blocks/hero-marquee/hero-marquee.css

```diff
@@ -0,0 +1,581 @@
+:root {
+  --s-min-height: 248px; /* 360px */
+  --m-min-height: 448px; /* 560px */
+  --l-min-height: 588px; /* 700px */
```

Comment:
Is this universally true from a Consonant perspective? If not, they might be better scoped to the `.hero-marquee` parent element.